libnavui-resources/src/main/res/drawable/mapbox_bg_circle.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@color/colorSurface" />
-    <corners android:radius="56dp" />
-</shape>
\ No newline at end of file
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file should not be changed since it is used by stable buttons `MapboxAudioButton`, `MapboxRecenterButton` and `MapboxOverviewButton`. Changing this file results in the following:

#### Before
<img src="https://user-images.githubusercontent.com/9770186/175675827-061b5709-c676-44da-9983-fce954343fae.jpg" width="250" />

#### After
<img src="https://user-images.githubusercontent.com/9770186/175676333-75769021-4f8b-42e5-a2c3-db04b9561cc4.jpg" width="250" />

We should rather add a new circular drawable `xml` to `drop-in ui` module since it has a different design for how circular buttons should look like.